projects/vendor/src/yotpo/yotpostarrating/yotpostarrating.component.html

```diff
@@ -0,0 +1,6 @@
+<ng-container *ngIf="(product$ | async) as product">
```

Comment:
Formatting will be fixed with `yarn prettier-fix`

Reply:
applied